packages/lodestar/src/chain/eventHandlers.ts

```diff
@@ -316,3 +317,11 @@ export async function onErrorBlock(this: BeaconChain, err: BlockError): Promise<
       break;
   }
 }
+
+function createCachedStateContext(stateContext: ITreeStateContext): ITreeStateContext {
```

Comment:
actually we can't completely remove epochProcess without any check bc `shouldUpdateJustifiedCheckpoint` maybe false in some case and it will only be updated in the next block, but we cleaned up in this case.